modules/cleaning-up-after-deleting-users.adoc

```diff
@@ -7,20 +7,20 @@
 
 After removing a user's access to {productname-long} or Jupyter, you must also delete their associated configuration files from {openshift-platform}.
 
-It is recommended that you back up the user’s data before removing their configuration files.
+Red Hat recommends that you back up the user’s data before removing their configuration files.
```

Comment:
Is "Red Hat recommends" appropriate for upstream?

Reply:
This file doesn't actually appear upstream. It is only included downstream.

Reply:
sorry about that!